src/adapters/rubicon.js

```diff
@@ -309,12 +309,65 @@ var RubiconAdapter = function RubiconAdapter() {
   }
 
   /**
+   * Register the default bidder settings for rubicon targeting
+   * @param {String} accountId
+   */
+  var _registerBidderSettings = (function() {
+    var _called = false;
+
+    // this function wrapped with closure to only run once
+    return accountId => {
+      if(!_called) {
+
+        bidmanager.registerDefaultBidderSetting(
+          RUBICON_BIDDER_CODE,
+          {
+            sendStandardTargeting: false,
+            suppressEmptyKeys: true,
+            adserverTargeting: [
+              'rpfl_' + accountId,
+              'rpfl_elemid'
+            ].map(key => ({
+                key: key,
+                val: bidResponse => bidResponse.rubiconTargeting && bidResponse.rubiconTargeting[key]
```

Comment:
We need a function here because bidResponse is not defined within this context, it is passed in later when prebid manages targeting for bid responses.  [Using a function here is a standard part of the prebid API though.](http://prebid.org/dev-docs/publisher-api-reference.html#adservertargeting)